app/presenters/projects_presenter.rb

```diff
@@ -22,6 +22,7 @@ def released_projects
   def projects
     @projects ||= begin
       query = Project.includes(:stages, snapshot: [:comparisons]).where(@params.permit(:organization_id))
+      query = query.where(criticality: @params[:criticality]) if @params[:criticality]
```

Comment:
You might be able to just add `:criticality` to the `permit` args in the line above and leave out this line.